### DISCUSSION TITLE
How do I downgrade Conemu to stable version.

### DISCUSSION THREAD

**Discussion by webmandman** [cmderdev/cmder#2539] [Q&A] [answered]
I use Ortus/Commandbox commandline tool that utilizes Jline and its having issues  with some minor things in the Cmder terminal. 

The author of commandbox suggested  I try a stable version of Conemu. So how  do I  get Cmder to use ConEmu 161206?

**Comment by daxgames** [comment, marked as answer]
1. Exit cmder.
2. RENAME `%cmder_root%\vendor\conemu-maximus5` folder  to something else.
3. Download whatever version you need.
4. Expand to `%cmder_root%\vendor\conemu-maximus5` folder.
5.  Run `cmder.exe`

FYI: The stable version is VERY old(5 years), last time I looked, and probably will work worse than what you have.
